layouts/partials/article/related.html

```diff
@@ -5,10 +5,11 @@ <h4 id="related-articles">Related articles</h4>
     <ul>
```

Comment:
Could we drop "articles" and just say "Related"?
Hugo's term "article" doesn't explicitly describe the type of doc content (task-based, reference, conceptual, and so on). The term especially stands out when the "article" isn't an entire page (as seems to be the intention in Hugo's structure along with "blogs" and "posts").

Reply:
Sure.